src/parch/mos/manifest.py

```
@@ -1,3 +1,11 @@
-from parch.compose.manifest import Manifest
+from typing import Any
 
 __all__ = ["Manifest"]
```

Ruff (0.16.2)

[error] Undefined name `Manifest` in `__all__`

(F822)